gravitee-apim-rest-api/gravitee-apim-rest-api-model/src/main/java/io/gravitee/rest/api/model/v4/plan/UpdatePlanEntity.java

```diff
@@ -80,8 +79,6 @@ public class UpdatePlanEntity {
 
     private String selectionRule;
 
-    private List<Flow> flows;
-
```

Comment:
I'm worried that changing this file could have repercussions in the old, non-onion part of the application. 
Perhaps we can use another class/create another class for all of the update information for a plan?